### PR TITLE
[12.0][FIX] web_widget_one2many_product_picker: Decimal precision

### PR DESCRIPTION
**Pull request by Tardo**
cc @tecnativa TT28298

### CODE REVIEW

File: web_widget_one2many_product_picker/models/ir_http.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Copyright 2020 Tecnativa - Alexandre D. Díaz
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl).
+from odoo import api, fields, models, tools
+from odoo.http import request
+
+
+class Http(models.AbstractModel):
+    _inherit = 'ir.http'
+
+    def get_currencies(self):
```
Comment (pedrobaeza) [inline web_widget_one2many_product_picker/models/ir_http.py]: This seems a very low level place to put this. Can you do it other way? You already have the currency in the form, and you can pass it as option to the widget if needed (but I don't think so, as the default is to `currency_id` if I'm not wrong.

Comment (Tardo) [reply web_widget_one2many_product_picker/models/ir_http.py]: Ok, now currencies are get from the widget.
Yes, by default uses "currency_id" and can be changed via options using "currency_field".

**Review by pedrobaeza — COMMENTED** [review]
How the monetary field gets this? I think we should mimic it and I don't think we need to dump the roundings for all the currencies.

**Comment by pedrobaeza** [comment]
I insist: can we avoid to retrieve all the currencies rounding. I guess the proper data should be around already thanks to monetary widget.

**Comment by Tardo** [comment]
Know the rounding value is necessary when the widget calculates the subtotal using the discount... we lost the server side rounding.

**Comment by pedrobaeza** [comment]
But how the monetary widget does it? I don't think that's a server operation.

**Comment by Tardo** [comment]
@pedrobaeza For example here, Monetary rounds the value to use in the SQL: https://github.com/odoo/odoo/blob/12.0/odoo/fields.py#L1411

**Review by pedrobaeza — APPROVED** [review]
OK, I see

**Comment by pedrobaeza** [comment]
Is this needed in v13, @Tardo ?

**Pull request closed without merging**